packages/core/src/session/index.ts

```diff
@@ -123,36 +123,3 @@ export function statelessSessions<Session> ({
     },
   }
 }
-
-/** @deprecated */
```

Comment:
@dcousens was this intentional? I can remove the documentation, for now I have moved this into redis example